osf_tests/test_registration_bulk_upload_parser.py

```diff
@@ -83,7 +91,7 @@ def provider_subjects(self):
 
     @pytest.fixture()
     def registration_provider(self, open_ended_schema, provider_subjects):
-        osf_provider = RegistrationProvider.get_default()
```

Comment:
Why did you do this? the _id should not be hard coded.

Reply:
Good question! I reverted this back to the same as the feature branch since I thought this meant to be in the scope of your provider PR. During my commit-by-commit rebase, this one has been changed several times so I went the safe way.

![Screen Shot 2022-07-15 at 2 30 19 PM](https://user-images.githubusercontent.com/3750414/179288624-dae70c65-73d2-4965-be46-9a4d43d2b303.png)